.ruby-version

```diff
@@ -1 +1 @@
-2.3.0
+2.3.4
```

Comment:
I have no idea.
As I checked, there are no problems.
All specs are still passed.

If it causes any problem,`.ruby-version` never cause-effect for gem users. So I think it couldn't be big problem.

Should I revert this changes?